PolarisServer/Client.cs

```diff
@@ -177,19 +187,29 @@ public void SendPacket(byte typeA, byte typeB, byte flags, byte[] data)
 
             Array.Copy(data, 0, packet, 8, data.Length);
 
-            SendPacket(packet);
+            SendPacket(packet, name);
         }
 
         public void SendPacket(Packet packet)
         {
             var h = packet.GetHeader();
-            SendPacket(h.Type, h.Subtype, h.Flags1, packet.Build());
+            SendPacket(h.Type, h.Subtype, h.Flags1, packet.Build(), packet.GetType().Name);
         }
 
         private void HandlePacket(byte typeA, byte typeB, byte flags1, byte flags2, byte[] data, uint position,
             uint size)
         {
-            Logger.Write("[-->] Packet {0:X}-{1:X} (flags {2}) ({3} bytes)", typeA, typeB, (PacketFlags)flags1, size + 8);
+            var handler = PacketHandlers.GetHandlerFor(typeA, typeB);
+            string packetName = "";
```

Comment:
Assigned value not used in any code path. Also recommend preferring `string.Empty`